app/views/shared/means_report/_outgoings_details.html.erb

```diff
@@ -0,0 +1,26 @@
+<section>
+  <% partner = @legal_aid_application.applicant.has_partner_with_no_contrary_interest? %>
```

Comment:
minor nit pick: this could be clearer as `partner?` to show it's a boolean check 
Further down it reads as partner object
```if partner && !partner_exclude_items.include?(item.value_method)```